src/components/LinkAsButton/index.tsx

```diff
@@ -6,24 +6,24 @@ import { NEW_PLAN_URL } from '../../constants';
 /** interface for LinkAsButton component props */
 export interface Props {
   classNameProp: string;
-  text: string;
+  link: string;
   to: string;
 }
 
 /** LinkAsButton component */
 const LinkAsButton = (props: Props) => {
-  const { to, classNameProp, text } = props;
+  const { to, classNameProp, link } = props;
   return (
     <Link to={to} className={classNameProp}>
-      {text}
+      {link}
     </Link>
   );
 };
```

Comment:
@ciremusyoka Why are we changing `props.text` to `props.link`?

It feels a little confusing b/c the actual "link" in this case is a URL which is passed in through `props.to`, whereas the child of `<Link />` is a "text" node so it may make more sense to keep that as `text`.

Seems to me like if anything props should change here it should be `to` => `link`.

@moshthepitt @p-netm what are your thoughts?

Reply:
My bad i miss understood the issue let me make the right changes.